src/test/java/org/broadinstitute/hellbender/utils/read/ReadUtilsIntegrationTest.java

```diff
@@ -48,8 +50,9 @@ public void testCreatePathSAMWriter(
       final boolean createMD5,
       final boolean expectIndex) throws Exception {
 
-    final Path outputPath = BucketUtils.getPathOnGcs(getGCPTestStaging() + "samWriterTest" + outputExtension);
-    final Path md5Path = BucketUtils.getPathOnGcs(getGCPTestStaging() + "samWriterTest" + outputExtension + ".md5");
+    final String randomizedOutputPathName = getGCPTestStaging() + "samWriterTest" + outputExtension + Integer.toString(new Random().nextInt());
```

Comment:
Unnecessary call to `Integer.toString()`, since `+` will automatically convert any object to a string in this context. (In general, I think `String.valueOf()` is preferable over `X.toString()` since it has the same behavior and doesn't need to be changed if the type of its argument ever changes.)

I'm not sure what this code is doing, but it seems like you'd want `outputExtension` after the random number, not before it.